auth/src/main/java/com/firebase/ui/auth/provider/PhoneProvider.java

```diff
@@ -34,8 +37,16 @@ public int getButtonLayout() {
 
     @Override
     public void startLogin(Activity activity) {
+
+        String phone = null;
+        for (AuthUI.IdpConfig idpConfig : mFlowParameters.providerInfo) {
+            if (idpConfig.getProviderId().equals(AuthUI.PHONE_VERIFICATION_PROVIDER)) {
+                phone = idpConfig.getParams().getString(EXTRA_DEFAULT_PHONE);
```

Comment:
To be consistent, we shouldn't statically import this and instead use `AuthUI.EXTRA_DEFAULT_PHONE`.

Reply:
Fixed